lib/govuk_admin_template.rb

```diff
@@ -19,4 +19,12 @@ def self.default_environment_style
       "development"
     end
   end
+
+  def self.google_analytics
+    @@google_analytics || Rails.env.production?
```

Comment:
But that means setting the `google_analytics` flag to false will have no affect?